x-worldpainter-world.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+    <mime-type type="application/x-worldpainter-world">
```

Comment:
`application/x-world-painter` would be better for consistency with the existing installers.